[2024-12-31T16:41:36.706+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.ft_balance_f manual__2024-12-31T16:40:58.613929+00:00 [queued]>
[2024-12-31T16:41:36.723+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.ft_balance_f manual__2024-12-31T16:40:58.613929+00:00 [queued]>
[2024-12-31T16:41:36.724+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-31T16:41:37.219+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_balance_f> on 2024-12-31 16:40:58.613929+00:00
[2024-12-31T16:41:37.237+0000] {standard_task_runner.py:60} INFO - Started process 1011 to run task
[2024-12-31T16:41:37.243+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'ft_balance_f', 'manual__2024-12-31T16:40:58.613929+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp5wxeip95']
[2024-12-31T16:41:37.248+0000] {standard_task_runner.py:88} INFO - Job 158: Subtask ft_balance_f
[2024-12-31T16:41:37.327+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.ft_balance_f manual__2024-12-31T16:40:58.613929+00:00 [running]> on host 403e7aadd7ee
[2024-12-31T16:41:37.446+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='ft_balance_f' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T16:40:58.613929+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-31T16:40:58.613929+00:00'
[2024-12-31T16:41:37.469+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-31T16:41:37.530+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-31T16:41:37.542+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=ft_balance_f, execution_date=20241231T164058, start_date=20241231T164136, end_date=20241231T164137
[2024-12-31T16:41:37.576+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-31T16:41:37.643+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
